packages/components/layout/src/Layout.tsx

```diff
@@ -37,7 +37,7 @@ const _Layout = (props: LayoutProps, ref: Ref<HTMLDivElement>) => {
     header,
     leftSidebar,
     rightSidebar,
-    variant = 'fullscreen',
+    variant = 'wide',
     className,
     testId = 'cf-layout',
```

Comment:
```suggestion
    testId = 'cf-ui-layout',
```

We missed the namespace here.